### ISSUE TITLE
Bug: Inconsistent Asciidoctor linting

### ISSUE THREAD

**Issue by srosenberg-coveo** [vale-cli/vale#258]
I'm using Vale on Mac OS, installed via Homebrew, version 2.5.1. Here is the same content in Markdown and Asciidoc, side by side. The rule I'm featuring in this example is meant to enforce capitalization for our product, component, and team names (although there's another issue we have with this rule which I've addressed separately [here](https://github.com/errata-ai/vale/issues/259)).

```
extends: substitution
message: "Use '%s' instead of '%s'."
ignorecase: false
level: error
action:
  name: replace
swap:
  ...
  "[wW]eb [sS]ource": Web source
  "[wW]eb [sS]ources": Web sources
```

![Screen Shot 2020-11-01 at 4 18 06 PM](https://user-images.githubusercontent.com/54185735/97815911-842dd880-1c5f-11eb-9a5a-32fc225938a8.png)

The Markdown file shows 21 errors, correctly flagging all of the incorrect occurrences, while the identical Asciidoc version only registers 16 of them. I don't really see a link, other than that all of the ignored occurrences are in the form of `Web Source`, although some occurrences of `Web Source` are being caught.

Do you have any idea what's going on here?

**Comment by jdkato** [comment]
Could you share the text in the AsciiDoc screenshot?

**Comment by srosenberg-coveo** [comment]
Certainly! I've removed the in-text configuration comments from the text below because those rules won't apply anyway.

```
= Module 1: Creating a Web Source

In this module, you'll create a web Source to index an employee site.

== Learning Objectives

By the end of this module you should:

* Have basic knowledge of what _web sources_ are.

* Be able to add and build a basic Web Source from the Administration Console.

== Lesson 1: Creating Web Sources

A *Web Source* is a named, persistent set of options that tells a _connector_ how and when to index original content from a given repository.

== Action 1: Add and Build a Web Source

Web source Web Source web source Web sources Web Sources web sources.

* Web source
* Web Source
* web source
* Web sources
* Web Sources
* web sources

|--
|Web source
|Web Source
|web source

|Web source
|Web Source
|web source

|Web sources
|Web Sources
|web sources
|--
```

**Comment by jdkato** [comment]
This will be fixed in the next release.

(although I think your table syntax should be `|===` instead of `|--`.)

**Comment by jdkato** [comment]
This should be fixed in `v2.6.0`.
